Dockerfile.local

```diff
@@ -11,7 +11,7 @@ RUN apk add --update tzdata
 FROM gcr.io/distroless/static-debian11:nonroot as distroless-static
 
 ############# builder-base #############
-FROM golang:1.18.5 AS builder-base
+FROM golang:1.19.0-alpine AS builder-base
```

Comment:
Why we are using alpine here?

Reply:
Because it's more lightweight and speeds up the initial build for `make gardener-up`. Full debian is not required for building